Makefile

```diff
@@ -44,9 +44,9 @@ apicast-source: ## Create Docker Volume container with APIcast source code
 	docker create --rm -v /opt/app-root/src --name $(COMPOSE_PROJECT_NAME)-source $(IMAGE_NAME) /bin/true
 	docker cp . $(COMPOSE_PROJECT_NAME)-source:/opt/app-root/src
 
-
-busted: dependencies $(ROVER) ## Test Lua.
-	@$(ROVER) exec bin/busted
+BUSTED_FILES ?=
+busted: dependencies $(ROVER) ## Test Lua. 
```

Comment:
Trailing whitespace.

Reply:
done in 3c177e4b05d3eb9af3868f8ad9710f6aa193e5ce